python/ml4ir/base/model/relevance_model.py

```diff
@@ -808,5 +809,47 @@ def calibrate(self, relevance_dataset, logger, logs_dir_local, temperature_init)
                                  dataset=relevance_dataset,
                                  logger=logger,
                                  logs_dir_local=logs_dir_local,
-                                 temperature_init=temperature_init,
-                                 file_io=self.file_io)
+                                 file_io=self.file_io,
+                                 **kwargs)
+
+    def add_temperature_layer(self, temperature: float = 1.0, layer_name: str = 'temperature_layer'):
```

Comment:
Just to double check, the default value temperature: float = 1.0 doesn't have any effect, right?

Reply:
A reasonable test ;-)

Reply:
But yes, jokes apart, it should have no effect. Although, thinking twice it is not a very silly test.

Reply:
@Ullimague  yes, T=1.0 has no effect because of  X/1.0==X

Reply:
It would be actually great to have such a test. And pretty trivial to add?

Reply:
I'm actually curious to know, if we train 2 same models (and with the same random seeds), one without temperature, and one with temperature T=1. Are the predictions the same?

Reply:
the prediction must be the same ; I added a test for temperature = 1.0 and compared the outputs of tho models (one with temperature =1.0 and the other without temperature) . Please see the test [here](https://github.com/salesforce/ml4ir/pull/119/commits/7fa0c7fcb7cb16b1c57c5a99144ef9e2a8d13903#diff-c604f786e9419883b3fdc3e0d13d6fa60b85ec7f2c8cb129dcd213abe5d7bac9R89)